controllers/root_controller.go

```diff
@@ -1,23 +1,35 @@
 package controllers
 
 import (
+	"../models/project"
 	"../modules/logging"
 	"github.com/flosch/pongo2"
 	"github.com/zenazn/goji/web"
 	"net/http"
+	"strconv"
 )
 
 type Root struct {
 }
 
 func (u *Root) Index(c web.C, w http.ResponseWriter, r *http.Request) {
-	_, err := LoginRequired(r)
+	current_user, err := LoginRequired(r)
```

Comment:
__[golint]__
* [golint] don't use underscores in Go names; var current_user should be currentUser